go.mod

```diff
@@ -8,6 +8,10 @@ replace cosmossdk.io/core => cosmossdk.io/core v0.11.0
 // breaks SDK app.toml parsing in ictest.
 replace github.com/spf13/viper => github.com/spf13/viper v1.17.0
 
+// TODO: not for production, seeing if removing .Jailed check panic from `ApplyAndReturnValidatorSetUpdates` fixes jailing issue.
+// https://github.com/Reecepbcups/cosmos-sdk/commit/26ff44b40ec78ca3e003aa49d9db1ce8199ae7dd
+replace github.com/cosmos/cosmos-sdk => github.com/Reecepbcups/cosmos-sdk v0.50.3-0.20240416135850-26ff44b40ec7
```

Comment:
TODO: remove me / get into the SDK (unless ofc there is another way around this)